websites/E/EternalNetwork TM/presence.ts

```diff
@@ -169,6 +170,40 @@ presence.on("UpdateData", async () => {
 
 			break;
 		}
+		case `et-log.${etrnl}`:
+		case `www.et-log.${etrnl}`: {
+			presenceData.smallImageKey = "eternallogistic_logo";
+
+			if (pathname.startsWith("/")) presenceData.state = ttl;
+
+			break;
+		}
+		case `dev.${etrnl}`:
+		case `www.dev.${etrnl}`: {
+			presenceData.smallImageKey = "eternalnetworktm_dev";
+
+			if (pathname.startsWith("/")) {
+				presenceData.state = "Developing new website or a features !";
+				presenceData.smallImageText = "Development & Features";
```

Comment:
Seems quite repetitive and elongated, please shorten this.

Reply:
Messages changed.